src/Analyzers/CSharp/Tests/PopulateSwitch/PopulateSwitchExpressionTests.cs

```diff
@@ -1040,7 +1040,7 @@ void Main()
             Bar.Option1 => 1,
             Bar.Option2 => 2,
             null => null,
-            _ => throw new System.NotImplementedException(),
+            Bar.Option3 => throw new System.NotImplementedException(),
```

Comment:
This test is misleading. It's named `TestAddMissingCasesForNullableEnum`, but the enum in it isn't nullable. Do you want me to make the enum nullable in it?

Reply:
dont' change existing tests.  you can rename them and/or add new tests.  thanks!

Reply:
@CyrusNajmabadi Well, I'll leave the test case and test name as-is for now. Only the expected result is updated.

Reply:
wfm.  feel free to update the name as well if you want.  but genrally don't make the test fit the name, make the name fit the test. and we feel there is a test hole, where no test checks the original name's case, then we can add that :)